tgui/packages/common/collections.ts

```diff
@@ -149,6 +149,8 @@ export const sortStrings = sortBy<string>();
 export const range = (start: number, end: number): number[] =>
   new Array(end - start).fill(null).map((_, index) => index + start);
 
+export const sort = sortBy();
```

Comment:
Is this ever actually used?

Reply:
Yeah, this constant thats just a call seems kinda useless

Reply:
addressed